conf/portal_modules.conf.defaults

```diff
@@ -16,6 +16,7 @@ modules=default_login_policy,default_guest_policy,default_oauth_policy,default_b
 [default_login_policy]
 description=Username/password login
 type=Authentication::Login
+source_id=local
```

Comment:
This makes it so that whatever "login type" sources (AD, LDAP, etc) you define on your connection profile ignored.

Reply:
Tell me if I'm wrong but when you use a connection profile with a root portal module that use the default login policy, sources defined on your connection profiles are not used.

I make this change because:
* all others authentication portal module has pre-defined sources
* it was unclear in the doc that you need to add a source on this specific portal module

Reply:
The sources on the connection profile are used if not sources are put in source_id

The Authentication::Login portal module encapsulates all the username/password sources, so that means you don't need to define all the source types or source IDs you want this to apply to unless you want to narrow it down to a specific source

Reply:
@julsemaan,

> The sources on the connection profile are used if not sources are put in source_id

Ok, I understand why I get some strange behaviors. We need to enhance documentation on that topic.

Reply:
> The sources on the connection profile are used if not sources are put in source_id

![screenshot from 2018-12-07 13-53-29](https://user-images.githubusercontent.com/10830260/49648806-7fd2f380-fa27-11e8-9465-fbf853bad159.png)

There is no help text that mentioned that the source on the connection profiles will be used.